internal/installation/receipt/receipt.go

```diff
@@ -44,6 +44,7 @@ func Load(path string) (index.Receipt, error) {
 	return indexscanner.ReadReceiptFromFile(path)
 }
 
+// receiptFromPlugin creates a receipt object using the plugin and index name.
 func receiptFromPlugin(indexName string, plugin index.Plugin) index.Receipt {
```

Comment:
Use capitalize Plugin in godoc 
Also let’s move Plugin to be first param. It’s likely we’ll add other params, so it’d better if it is first parameter.